### ISSUE TITLE
CryptoKitties is blocked

### ISSUE THREAD

**Issue by hankingithub** [MetaMask/eth-phishing-detect#550]
Please unblock access to CryptoKitties.

**Comment by 409H** [comment]
Which domain are you looking at?

The legit domain is cryptokitties.co and it's whitelisted.

**Comment by hankingithub** [comment]
Its working now. I don't what was happening.

**Comment by 409H** [comment]
Ok - Will close the issue. If you encounter another issue please open a new issue and provide screenshots and details of the behaviour.
